governance/policies/settings-policy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "actions": [
+        {
+            "name": "set_settings_policy",
+            "args": {
+                "settings_policy": {
+                    "service": {
+                        "name": "KMS",
+                        "description": "Key Management Service",
+                        "version": "1.0.0",
+                        "debug": true
```

Comment:
If this file is intended as the debug setting, how about renaming it to `settings-policy-debug.json` or something like that. 
We want to avoid it's used for production by accident.

Reply:
no this will be way more in the future than only debugging.